src/util/buffer.h

```diff
@@ -23,6 +23,7 @@ Revision History:
 
 #include <type_traits>
 #include "util/memory_manager.h"
+#include "debug.h"
 
```

Comment:
why is it here?

Reply:
For some magical reason, suddenly, some random file within Z3 started giving errors about unimplemented macro SASSERT. No idea why and no idea how to solve it. I can try to debug further, though.

Reply:
ok

Reply:
I would like to resolve this mysterious anomaly, but for the sake of merging the PR, it might not be now if you are OK with that.